server/controllers/admin/enterprises.js

```diff
@@ -171,3 +172,21 @@ exports.update = function update(req, res, next) {
     .catch(next)
     .done();
 };
+
+// POST /enterprises/:id/logo
+exports.uploadLogo = (req, res, next) => {
+  if (req.files.length === 0) {
+    next(BadRequest('Expected at least one file upload but did not receive any files.'));
+    return;
+  }
+
+  const logo = req.files[0].link;
+  const sql = 'UPDATE enterprise SET logo = ? WHERE id = ?';
+
+  db.exec(sql, [logo, req.params.id])
+    .then(() => {
+      res.status(200).json({ link : logo });
```

Comment:
I think this should just be `res.status(200).json({ logo })`.  Since we call the property "logo" in the table, we should return that name here.